app/cronjob/trailingTrade/step/ensure-grid-trade-order-executed.js

```diff
@@ -254,6 +255,13 @@ const execute = async (logger, rawData) => {
         },
         temporaryDisableActionAfterConfirmingOrder
       );
+
+      // Queue other symbols to check if max. open trade is reached
+      symbols.map(async symbolToQueue => {
```

Comment:
`map` can be replaced with `forEach` and I think no need for `async` here because we are not expecting a result.